packages/addons/tools/jre.zulu/source/default.py

```diff
@@ -0,0 +1,2 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
```suggestion
# SPDX-License-Identifier: GPL-2.0
```

Reply:
to align with all other default.py files we have, because there is no copy worthy content here so just a placeholder? also this is addon is introduced 2019 so i guess 2019-present ?